cmd/cloud/server.go

```diff
@@ -49,6 +49,8 @@ func init() {
 	serverCmd.PersistentFlags().Bool("cluster-installation-supervisor", true, "Whether this server will run a cluster installation supervisor or not.")
 	serverCmd.PersistentFlags().String("state-store", "dev.cloud.mattermost.com", "The S3 bucket used to store cluster state.")
 	serverCmd.PersistentFlags().StringSlice("allow-list-cidr-range", []string{"0.0.0.0/0"}, "The list of CIDRs to allow communication with the private ingress.")
+	serverCmd.PersistentFlags().StringSlice("vpn-list-cidr", []string{}, "The list of VPN CIDRs to allow communication with the clusters.")
```

Comment:
Is it possible to set a default here that is dev friendly like we do above with `allow-list-cidr-range`?